### DISCUSSION TITLE
play NES save file on emulator

### DISCUSSION THREAD

**Discussion by baron-v-hinten** [sanni/cartreader#1049] [Need help]
hey guys,
I have just received my cartreader and one of the first dumps I created with it was a copy of "The Legend of Zelda" on NES. I also wanted to get the savegames, which made the device create a *.bin file on my SD card. My question is: How can I get this to run on an emulator? I tried two of them and both didn't seem to be able to open that file.

In order to double-check, I also dumped my SNES copy of "Zelda - Link to the Past" as well as its save files, copied them into a folder, named both (the rom and the savegame) exactly the same and the emulator was able to recognise the *.srm savefile and immediately loaded it correctly into the game. So the process works for other consoles.

I would be thankful for any kind of help / advice, thank you! =)

**Comment by Ryo-Seadas** [comment]
I was about to ask the same question. Usually Nes emulators use .sav as format for savegames.
I couldn't find any converter online. I'm interested to use my original saves on emulators too.

**Comment by nsx0r** [comment]
Hi,

Usually you have to rename the save file from *.bin to *.sav and place it in your emulator save folder, for example:
- FCEUX: \fceux\sav\
- MEsen: \Mesen\Saves\
- Nintendulator: %appdata%\Nintendulator\SRAM\
 
The emulator will load the save file automatically if the filenames match between the ROM file and the SAVE file.
Tested working with the 3 emulators listed above.

**Comment by Ryo-Seadas** [comment]
Thank you very much. I tried to rename the save from .bin to .sav, but didn't work. After I found out that on the original cart the saves where gone 😅. Probably the battery died meantine 😅

**Comment by baron-v-hinten** [comment]
Thank you, nsx0r, for your reply! Didn't think it was that easy - can actually confirm the success, tested it on Mesen.
One last question: have you ever tried renaming a sav game you continued on an emulator back into a bin file and then write it back onto the cartridge, i.e. does it also work the other way round?

Thank you again and all the best for 2025! :)

**Reply by Ryo-Seadas** [reply]
@baron-v-hinten I tried it with a downloaded save file for Zelda 2 that I was using on a emulator. Written on the original cart, it was a success
